### PR TITLE
Add CDC Monkeypox scraper

### PR DESCRIPTION
**Pull request by smcclure17**
Add a scraper to pull in the cumulative Monkeypox cases reported by the CDC. 

This ingests the Monkeypox data in the existing Covid schema/infrastructure without any distinction (beyond the variable name) which is something I think we'll want to clean up if we decide to further invest in Monkeypox data (see my comments below) but until then I'm not sure it's worth the effort.

### CODE REVIEW

File: can_tools/bootstrap_data/covid_categories.csv [hunk 1]
```diff
@@ -1,5 +1,6 @@
 group,category
 case,cases
+case,monkeypox_cases
```
Comment (smcclure17) [inline can_tools/bootstrap_data/covid_categories.csv:3]: This does stray us a bit away from the intention of this schema and I'm not a huge fan of doing it this way, but: 

Ideally, I think we'd want to spin up a new schema that's parallel to the `covid` schema for this monkeypox data (which would consist of creating a `monkeypox_observations` table and a `monkeypox_us` view in Postgres, or something similar), but I'm not sure that's worth the investment at the moment.

Comment (smcclure17) [reply can_tools/bootstrap_data/covid_categories.csv:3]: Those schemas are created here: https://github.com/covid-projections/can-scrapers/blob/main/can_tools/models.py#L219

Comment (mikelehen) [reply can_tools/bootstrap_data/covid_categories.csv:3]: Yeah, I think this is the right approach for now.  If we get serious about collecting monkeypox data (e.g. scraping state dashboards and/or related monkeypox metrics), then we should revisit.

**Review by mikelehen — APPROVED** [review]
LGTM!  Do we need to add this to the `ACTIVE_SCRAPERS` list?

**Comment by smcclure17** [comment]
> Do we need to add this to the ACTIVE_SCRAPERS list?

Ah, right 🤦 thanks!

**Pull request merged**